packages/react-router/modules/matchPath.js

```diff
@@ -1,5 +1,29 @@
 import pathToRegexp from "path-to-regexp";
 
+const isAbsolute = pathname => !!(pathname && pathname.charAt(0) === "/");
```

Comment:
```javascript 
const isAbsolute = pathname => /^\\?\/.test(pathname);
```

This will make all the tests pass - pathname may come already escaped with "\"

Reply:
Thanks for the tip; while your snippet isn't valid (unclosed regex), it pointed me in the right direction. Changes have been made and tests are passing.